GoogleSheetI18n/Integrations/Web/I18nController.cs

```diff
@@ -86,5 +101,51 @@ public object Get()
                 _i18nOptions.SubscriptionUrl
             };
         }
+
+        private object GetWarnings(IList<I18nSheet> i18nSheets)
```

Comment:
Looks like, this method shouldn't be here.

Reply:
removed